### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by OMatiykiv**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+.block-5 {
+  grid-area: more;
+  height: 300px;
+  margin: 1vmin 0.5vmin;
+  text-align: center;
+  background-color: brown;
+}
+.block-6 {
+  grid-area: aside;
+  margin: 1vmin 0.5vmin;
```
Comment (karmeljuk) [inline css/styles.css]: .5vmin

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+  color: black;
+  text-decoration: none;
+}
+nav a:hover {
+  transform: scale(1.2);
+  color: #b8b8b8;
+}
+.block-7 {
+  min-height: 50px;
+  margin: 1vmin 0.5vmin;
```
Comment (karmeljuk) [inline css/styles.css]: .5vmin

**Review by karmeljuk — CHANGES_REQUESTED** [review]
great job. Minor changes needed

File: css/styles.css (hunk 1 above)
Comment (OMatiykiv) [reply css/styles.css]: done

File: css/styles.css (hunk 2 above)
Comment (OMatiykiv) [reply css/styles.css]: done

**Review by karmeljuk — APPROVED** [review]

**Pull request merged**